.travis.yml

```diff
@@ -1,8 +1,11 @@
 
 language: node_js
+os:
+  - linux
+  - osx
 node_js:
-  - "5"
+  - "8"
```

Comment:
Ah, lovely, so it is supported after all. I remember bumping into this a few weeks before (can't remember exactly where) and apparently `8` was not supported back then.